transport/processing/processors.py

```diff
@@ -21,13 +21,13 @@ def process(self, processing_job):
 
 class FileFilter(ProcessorBase):
     def __init__(
-        self,
-        filter_files,
+            self,
```

Comment:
nit: are you sure the additional indentation is "correct"? I would have thunk it was better before..

Reply:
I was a bit surprised as well but this is in PEP 8, basically we need an extra level of indentation to distinguish arguments from the rest.

Reply:
hmm, okay, if pep8 says so, I shall not question this :-)